client/components/public/Gear.jsx

```diff
@@ -246,40 +244,20 @@ Gear = class Gear extends Component {
                   <List.Item>
                     <strong>Domestic Shipping</strong>:
                     <List.List>
-                      <List.Item>For any T-shirt (long or short sleeve):</List.Item>
-                      <List.List>
-                        <List.Item>1 shirt: $5</List.Item>
-                        <List.Item>2&ndash;3 shirts: $8</List.Item>
-                        <List.Item>4&ndash;6 shirts: $15</List.Item>
-                      </List.List>
-                      <List.Item>For any sweatshirts (crew, hoodie, &frac14; zip)
-                        <List.List>
-                          <List.Item>1 sweatshirt: $8</List.Item>
-                          <List.Item>2 sweatshirts: $15</List.Item>
-                        </List.List>
-                      </List.Item>
+                      <List.Item>For any T-shirt (long or short sleeve): $5/shirt</List.Item>
+                      <List.Item>For any sweatshirts (crew, hoodie, &frac14; zip): $8/sweatshirt</List.Item>
                     </List.List>
                   </List.Item>
                   <List.Item>
                     <strong>International Shipping</strong>:
                     <List.List>
-                      <List.Item>For any T-shirt (long or short sleeve):</List.Item>
-                      <List.List>
-                        <List.Item>1&ndash;3 shirts: $25</List.Item>
-                        <List.Item>4&ndash;6 shirts: $50</List.Item>
-                      </List.List>
-                      <List.Item>For any sweatshirts (crew, hoodie, &frac14; zip)
-                        <List.List>
-                          <List.Item>1 sweatshirt: $25</List.Item>
-                          <List.Item>2 sweatshirts: $50</List.Item>
-                        </List.List>
-                      </List.Item>
+                      <List.Item>For any shirt (T-shirt or sweatshirt): $25/shirt</List.Item>
                     </List.List>
                   </List.Item>
                 </List>
               </Grid.Column>
               <Grid.Column width={5}>
-                  <Image src='https://gph-distributed.s3.us-west-2.amazonaws.com/2022/gear/shirt_design.png'/>
+                  <Image src='https://gph-distributed.s3.us-west-2.amazonaws.com/2023/gear/shirt_design.png'/>
```

Comment:
Should this reference the s3_prefix const defined above?

Reply:
Good catch! Updated just now.